system-tests/README.md

```diff
@@ -1,122 +1,260 @@
+> For Example, the `Registration Service`, `Identity Hub` and `EDC` repository hash can be found in the _Checkout_ steps  (in the `ref` property) of [action.yml](./.github/actions/../../../.github/actions/gradle-setup/action.yml):
+
+```yml
+    - name: Checkout EDC
+      uses: actions/checkout@v2
+      with:
+        repository: eclipse-dataspaceconnector/DataSpaceConnector
+        path: DataSpaceConnector
+        ref: 3ff940b720f44826df28e893fb31344eb6faacef
+
+    - name: Checkout Registration Service
+      uses: actions/checkout@v2
+      with:
+        repository: eclipse-dataspaceconnector/RegistrationService
+        path: RegistrationService
+        ref: 374c14bcca23ddb1dcd7476a27264510e54de7fa
+
+    - name: Checkout Identity Hub
+      uses: actions/checkout@v2
+      with:
+        repository: eclipse-dataspaceconnector/IdentityHub
+        path: IdentityHub
+        ref: bc13cf0cb8589b792eef733c7cf7b3422476add5
 
-The test copy a file from provider to consumer blob storage account.
+```
+
+> After you have cloned the `EDC`, `Identity Hub` and `Registration Service` repos locally you should run the command to
+> `reset` to the specific hash.  
+>
+> For Example:
+
+```bash
+# EDC (in the EDC root folder)
+git reset --hard 3ff940b720f44826df28e893fb31344eb6faacef
+
+# Identity Hub (in the Identity Hub root folder)
+git reset --hard bc13cf0cb8589b792eef733c7cf7b3422476add5
+
+# Registration Service (in the Registration Service root folder)
+git reset --hard 374c14bcca23ddb1dcd7476a27264510e54de7fa
+```
+
+> Now you can follow the rest of the process below.  
+> Once the publications are available in _Maven Central_ this process will not be necessary
+> 
+<br />
+
+### EDC
+
+<br />
+
+Execute the following command  from `EDC` root folder.
+
+```bash
+./gradlew publishToMavenLocal -Pskip.signing
+```
+
+_Note for Windows PowerShell, the following command should be used:_
+
+```powershell
+./gradlew publishToMavenLocal -P"skip.signing"
+```
+
+<br />
+
+### Identity Hub
+
+<br />
+
+Execute the following command from `Identity Hub` root folder:
+
+
+```bash
+./gradlew publishToMavenLocal -Pskip.signing
+```
+
+_Note for Windows PowerShell, the following command should be used:_
+
+```powershell
+./gradlew publishToMavenLocal -P"skip.signing"
+```
 
-### Building MVD project
+<br />
 
-MVD dependencies are Eclipse DataSpaceConnector(EDC) and Registration Service. Both of these dependencies are not published to any central artifactory yet so in local
-development we have to use locally published dependencies, once this is done MVD can be build using
+### Registration Service
+
+<br />
+
+Execute the following command from `Registration Service` root folder:
+
+```bash
+./gradlew publishToMavenLocal
+```
+
+<br />
+
+### MVD
+
+<br />
+
+Now that the publishing to the local repositories has been completed, `MVD` can be built by running the following command from the root of the `MVD` project folder:
 
 ```bash
 ./gradlew build -x test
```

Comment:
why should tests be skipped?

Reply:
Otherwise integration tests will fail as docker is still not started as per instructions.

If we want to run it with test then all integration tests needs to be disabled first by default. (Can be a seperate PR)

Reply:
not making any change to this PR for this; as @cpeeyush indicated seperate PR could address